pkg/distributor/ha_tracker.proto

```diff
@@ -22,4 +22,6 @@ message ReplicaDesc {
     // already remove entry from memory. Actual deletion from KV store does *not* trigger
     // "watch" notification with a key for all KV stores.
     int64 deleted_at = 3;
+    // This is the most recent timestamp when this replica was elected as the leader.
+    int64 elected_at = 4;
```

Comment:
I was wondering if we can also add an `int64 version` field here, to be incremented on each replica change (but not on received_at update). Then we can expose that field as a gauge, and we'd be able to measure the rate of the HA reelections with total precision.